app/server/src/main/scala/org/bitcoins/server/BitcoindRpcBackendUtil.scala

```diff
@@ -211,7 +193,8 @@ object BitcoindRpcBackendUtil extends Logging {
       interval: FiniteDuration = 10.seconds)(implicit
       system: ActorSystem,
       ec: ExecutionContext): Cancellable = {
-    val atomicPrevCount: AtomicReference[Int] = new AtomicReference(startCount)
+    val numParallelism = Runtime.getRuntime.availableProcessors()
```

Comment:
This function is probably fine to leave as is, it is only being called every 10 seconds

Reply:
Hmm... so you are saying that the default interval we provide is too fast (`10.seconds`). I think you might be right, as if there is a old wallet that hasn't synced in a long time, we could start syncing twice (if the first sync takes > 10 seconds)?

Reply:
Yeah, this could be a problem.

Reply:
Do you have a recommendation of what to change it to? Like 5 minutes?

Reply:
I think what might be better is to leave this as it, but only start the polling after the intial sync is done

Reply:
as per conservation with ben, we are going to address this in #2926